pandas/io/clipboard/clipboards.py

```diff
@@ -1,6 +1,6 @@
 import subprocess
 
-from pandas.compat import PY2, text_type
+from pandas.compat import text_type
```

Comment:
Just use str?

Reply:
Yep, we should probably be able to do that.